tempesta_fw/sock.c

```diff
@@ -72,7 +72,7 @@ ss_ipi(struct irq_work *work)
 }
 
 static int
-ss_wq_push(SsWork *sw)
+ss_wq_push(SsWork *sw, bool sync)
```

Comment:
Maybe it's C++'ism, but `bool` isn't `int`: please use `0` for int arguments or `true`/`false` for boolean, but don't intermix them.